src/common/grealpath.h

```diff
@@ -20,10 +23,48 @@ static inline gchar *g_realpath(const char *path)
 #define PATH_MAX 4096
 #endif
   char buffer[PATH_MAX] = { 0 };
-  if(realpath(path, buffer))
+  char* res = realpath(path, buffer);
+
+  if(res) 
+  {
     return g_strdup(buffer);
-  else
-    return NULL;
+  }
+  else 
+  {
+    if ( errno == EACCES ) 
+    {
+      printf("Read or search permission was denied for a component of file_name: %s\n", path);
```

Comment:
Likewise, printf() should be fprintf on stderr. All printf below too. + translatable.